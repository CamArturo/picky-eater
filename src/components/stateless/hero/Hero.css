.hero-container {
  height: 500px;
  width: 100%;
  background-image: url("../../../assets/restaurant-food.jpg");
  display: flex;
  align-items: center;
  justify-content: center; }
  .hero-container .inner {
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    align-items: center; }
    .hero-container .inner h2 {
      font-family: Signika Negative, serif;
      font-style: normal;
      font-weight: bold;
      line-height: 56px;
      font-size: 36px;
      letter-spacing: 0.04em;
      margin: 0;
      color: #FCFBFB;
      text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.7); }
    .hero-container .inner button {
      width: auto;
      height: 40px;
      font-family: Catamaran, serif;
      font-weight: bold;
      font-size: 18px;
      line-height: 37px;
      letter-spacing: 2px;
      color: #fff;
      background-color: #0956B2;
      border: none;
      margin: 0 0 0 1.5rem; }
      .hero-container .inner button:hover {
        color: #0956B2;
        background-color: #fff;
        cursor: pointer; }

/* Large Devices, Wide Screens */
/* Medium Devices, Desktops */
/* Small Devices, Tablets */
/* Extra Small Devices, Phones */
/* Custom, iPhone Retina */

/*# sourceMappingURL=Hero.css.map */
