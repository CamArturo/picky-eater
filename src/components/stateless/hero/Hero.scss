.hero-container {
  height: 500px;
  width: 100%;
  //margin-bottom: 50px;

  background-image: url("../../../assets/restaurant-food.jpg");

  display: flex;
  align-items: center;
  justify-content: center;
  .inner {
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    align-items: center;
    h2 {
      font-family: Signika Negative, serif;
      font-style: normal;
      font-weight: bold;
      line-height: 56px;
      font-size: 36px;
      letter-spacing: 0.04em;
      margin: 0;

      color: #FCFBFB;
      text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.7);
    }
    button {
      width: 184px;
      height: 40px;
      font-family: Catamaran, serif;
      font-weight: bold;
      font-size: 18px;
      line-height: 37px;
      letter-spacing: 2px;
      color: #fff;
      background-color: #0956B2;
      border: none;
      margin: 0 0 0 1.5rem;
    }
  }
}

/* Large Devices, Wide Screens */
@media only screen and (max-width: 1200px) {

}

/* Medium Devices, Desktops */
@media only screen and (max-width: 992px) {

}

/* Small Devices, Tablets */
@media only screen and (max-width: 768px) {

}

/* Extra Small Devices, Phones */
@media only screen and (max-width: 480px) {

}

/* Custom, iPhone Retina */
@media only screen and (max-width: 320px) {

}