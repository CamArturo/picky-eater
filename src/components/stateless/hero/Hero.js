import React from 'react';
import { NavLink, Route, withRouter } from 'react-router-dom';
import './Hero.css';

export const Hero = () => (
  <section className="hero-container">
    <section className="inner">
      <h2>Find it. Finally.</h2>
      <NavLink to="/location">
        <button>Restaurant Match</button>
      </NavLink>
    </section>
    <Route exact path='/location' component={Location} />
  </section>
);

export default withRouter(Hero);