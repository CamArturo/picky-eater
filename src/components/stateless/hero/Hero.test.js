import React from 'react';
import { Hero } from './Hero';
import { shallow } from 'enzyme';


it('Should match the snapshot', () => {
  const appComponent = shallow(<Hero />);

  expect(appComponent).toMatchSnapshot();
});
