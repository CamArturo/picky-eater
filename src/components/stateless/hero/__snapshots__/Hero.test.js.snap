// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <section
          className="inner"
>
          <h2>
                    Find it. Finally.
          </h2>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/location"
          >
                    <button>
                              Restaurant Match
                    </button>
          </NavLink>
</section>,
        <Route
          component={[Function]}
          exact={true}
          path="/location"
/>,
      ],
      "className": "hero-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              Find it. Finally.
</h2>,
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              to="/location"
>
              <button>
                            Restaurant Match
              </button>
</NavLink>,
          ],
          "className": "inner",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Find it. Finally.",
            },
            "ref": null,
            "rendered": "Find it. Finally.",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": <button>
                Restaurant Match
</button>,
              "to": "/location",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Restaurant Match",
              },
              "ref": null,
              "rendered": "Restaurant Match",
              "type": "button",
            },
            "type": [Function],
          },
        ],
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/location",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section
            className="inner"
>
            <h2>
                        Find it. Finally.
            </h2>
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        to="/location"
            >
                        <button>
                                    Restaurant Match
                        </button>
            </NavLink>
</section>,
          <Route
            component={[Function]}
            exact={true}
            path="/location"
/>,
        ],
        "className": "hero-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                Find it. Finally.
</h2>,
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/location"
>
                <button>
                                Restaurant Match
                </button>
</NavLink>,
            ],
            "className": "inner",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Find it. Finally.",
              },
              "ref": null,
              "rendered": "Find it. Finally.",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": <button>
                  Restaurant Match
</button>,
                "to": "/location",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Restaurant Match",
                },
                "ref": null,
                "rendered": "Restaurant Match",
                "type": "button",
              },
              "type": [Function],
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/location",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
